src/Middleware.js

```diff
@@ -0,0 +1,63 @@
+import {KEYFRAMES, DECLARATION} from './Enum.js'
+import {strlen} from './Utility.js'
+import {stringify} from './Serializer.js'
+import {prefix} from './Prefixer.js'
+
+/**
+ * @param {object} element
+ * @param {function} callback
+ * @return {string}
+ */
+export function prefixer (element, callback) {
+	switch (element.type) {
+		case DECLARATION: return prefix(element.value, strlen(element.props))
+		case KEYFRAMES: return prefix(stringify(element, callback), 10)
+	}
+
+	return stringify(element, callback)
+}
+
+/**
+ * @param {function} callback
+ * @return {function}
+ */
+export function rulesheet (callback) {
```

Comment:
shouldnt this act more like a `middleware`? in the sense that it should accept plugins? `prefixer` shouldnt be included in here by default, right? we should also be able to provide other plugins as well

Reply:
Yes, i've stumbled on an implementation that would allow the previous `middlware([prefixer, stringify, rulesheet])` design to work with this type of plugin.

Reply:
Eager to see what you have come up with!